src/ai/backend/manager/services/session/service.py

```diff
@@ -1317,6 +1317,7 @@ async def get_session_info(self, action: GetSessionInfoAction) -> GetSessionInfo
         resp["tag"] = sess.tag
 
         # Resource occupation
+        resp["kernelId"] = str(sess.main_kernel.id)
```

Comment:
Single session can have multiple kernels.
We need to change the kernel loading conditions and retrieve all kernels.